CHANGELOG.rst

```diff
@@ -17,11 +17,16 @@
   is specified on the command line together with the ``--pyargs``
   option.  Thanks to `@taschini`_ for the PR (`#1597`_).
 
+* Rename ``getfuncargvalue`` to ``getfixturevalue``. `getfuncargvalue`
+  deprecated but still present. Thanks to `@RedBeardCode`_ and `@tomviner`_
```

Comment:
The linting fails because we added @RedBeardCode to Changelog links in another PR.